css.md

```diff
@@ -137,3 +138,15 @@ $_componentNameColor: #000;
 ```scss
 $_componentNameColor: $color1;
 ```
+
+### [2.3](#scss--comments): Comments
+
+When commenting in sass be sure to use `//` version of comments this version will be stripped out during the compilation procress
```

Comment:
just a grammar fix: "...version of comments. Comments written in this style are stripped out during the compilation process."

Reply:
Updated branch based on this comment